src/init.cpp

```diff
@@ -293,6 +286,14 @@ void Shutdown(NodeContext& node)
     if (node.mempool) node.mempool = nullptr;
     node.scheduler.reset();
     LogPrintf("%s: done\n", __func__);
```

Comment:
I would put line 288 at the very end.
Done should always be the last message in a log. Otherwise it would be weird if stuff still follows

Reply:
[Done](https://github.com/bitcoin/bitcoin/pull/18526#issuecomment-609823729).